driver/random.hpp

```diff
@@ -1,16 +1,26 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
-#include <cstdlib>
+#include <random>
+
+std::minstd_rand& get_minstd_gen()
+{
+    static thread_local std::minstd_rand minstd_gen(std::random_device{}());
+    return minstd_gen;
+}
 
 template <typename T>
 inline T FRAND()
 {
-    double d = static_cast<double>(rand() / (static_cast<double>(RAND_MAX)));
+    double d = std::generate_canonical<double, 1>(get_minstd_gen());
     return static_cast<T>(d);
 }
 
-inline int GET_RAND() { return rand(); }
+inline int GET_RAND()
```

Comment:
This breaks the verification cache feature (`--verification_cache` or `-C`) in the convolution driver. The reason is that results of `GEN_RAN` is not reproducible anymore.